hbase-shell/src/main/ruby/shell/commands/get_namespace_rsgroup.rb

```diff
@@ -0,0 +1,38 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+module Shell
+  module Commands
+    class GetNamespaceRsgroup < Command
+      def help
+        <<-EOF
+Get the group name the given NameSpace is a member of.
+
+Example:
+
+  hbase> get_namespace_rsgroup 'namespace_name'
+
+EOF
+      end
+
+      def command(namespace_name)
+        group_name = admin.get_namespace_rsgroup(namespace_name)
```

Comment:
How will `group_name` be printed? I think you just removed print statement.

Reply:
![image](https://user-images.githubusercontent.com/21099711/104282434-02bdb680-54ea-11eb-8e06-5c5ee2e0a3db.png)

I code it from get_server_rsgroup method, and it look good !

Reply:
With same changes, can you try searching for namespace which doesn't have any rsgroup?
e.g if namespace 'NMn' exists but it is not part of any rsgroup (rsgroup feature is not enabled on cluster), then what will `get_namespace_rsgroup 'NMn'` return?

Reply:
thanks a lot， I will update this pr